pkg/projects/Microsoft.NETCore.App/Microsoft.NETCore.App.pkgproj

```diff
@@ -64,6 +64,11 @@
       <TargetFramework>.NETCoreApp1.1</TargetFramework>
     </Dependency>
 
+    <!-- references the host packages -->
+    <Dependency Include="Microsoft.NETCore.DotNetHostPolicy">
```

Comment:
will remove the files from sharedFrameworkPublish as part of followup PR, when sdk is stabilized